hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -1023,7 +1024,10 @@ cdef class PureMarketMakingStrategy(StrategyBase):
             f"({limit_order_record.quantity} {limit_order_record.base_currency} @ "
             f"{limit_order_record.price} {limit_order_record.quote_currency}) has been completely filled."
         )
+        buyTimestamp = self._current_timestamp
+        buyDateTime = datetime.fromtimestamp(buyTimestamp)
```

Comment:
How about condensing these two lines (and removing import of datetime) by using:
pd.Timestamp.fromtimestamp(self._current_timestamp)

Please try it out and if successful I will approve this code